AUDIO/N_0xf1f8157b8c3f171c.md

```diff
@@ -1,17 +1,18 @@
 ---
 ns: AUDIO
 ---
-## _0xF1F8157B8C3F171C
+## SET_VEHICLE_STARTUP_REV_SOUND
 
 ```c
 // 0xF1F8157B8C3F171C 0x1C0C5E4C
-void _0xF1F8157B8C3F171C(Vehicle vehicle, char* p1, char* p2);
+void SET_VEHICLE_STARTUP_REV_SOUND(Vehicle vehicle, char* p1, char* p2);
 ```
 
 ```
```

Comment:
Please convert this into a proper description (without a code block) and add the examples (with the new native name) as described here: https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md